src/Microsoft.Tye.Core/ProjectReader.cs

```diff
@@ -180,8 +180,10 @@ private static void EvaluateProject(OutputContext output, DotnetProjectServiceBu
             output.WriteDebugLine($"Found target framework: {targetFramework}");
 
             // TODO: Parse the name and version manually out of the TargetFramework field if it's non-null
-            project.TargetFrameworkName = GetMetadataValueOrEmpty("_ShortFrameworkIdentifier");
+            project.TargetFrameworkName = GetMetadataValueOrNull("_ShortFrameworkIdentifier") ?? project.TargetFramework.TrimEnd(".0123456789".ToCharArray());
```

Comment:
@davidfowl Looks like you were the original author for the use of the `_ShortFrameworkIdentifier` property. This property isn't resolved given the latest 5.0 SDK. What was setting this?

I've added a fallback so 5.0 SDK doesn't break the resolution of the TFM name. cc @jkotalik for review.